src/pow_sse.c

```diff
@@ -475,7 +476,7 @@ ImplContext PoWSSE_Context = {
     .context = NULL,
     .description = "CPU (Intel SSE)",
     .bitmap = 0,
-    .num_max_thread = 2,
+    .num_max_thread = 0,
```

Comment:
Would you consider to revise its name from `num_max_thread` to `max_thread_num` or `max_thread_per_cpu`?

Reply:
Yes.
By the way, the variable name always makes me confused to another similar variable name `num_max_threads` in `pow_avx.h`.
It should be renamed.

Reply:
In addition, clarify in the entries in directory `docs/`.